assembl/static2/js/app/components/cookies/cookiesSelector.jsx

```diff
@@ -0,0 +1,96 @@
+// @flow
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+import { Button } from 'react-bootstrap';
+import classnames from 'classnames';
+import CookieToggle from './cookieToggle';
+import type { CookieObject } from './cookieToggle';
+
+type CookiesSelectorProps = {};
+
+type CookiesSelectorState = {
+  activeKey: ?string,
+  show: boolean,
+  cookies: Array<CookieObject>
+}
+
+
+class CookiesSelector extends React.Component<CookiesSelectorProps, CookiesSelectorState> {
+  constructor(props: CookiesSelectorProps) {
+    super(props);
+    const cookiesList = document.cookie.split(' ');
+    const cookies = [];
+    cookiesList
+      .map(cookie => ({ ...this.getCookieObject(cookie), accepted: true }))
+      .forEach(cookieObject => cookies.push(cookieObject));
```

Comment:
why not `const cookies = cookiesList.map(cookie => ({ ...this.getCookieObject(cookie), accepted: true }))` ?